shared/persist-artefacts.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+ARGV_DIRECTORY="$1"
+set -u
+
+mkdir -p $(pwd)/${ARGV_DIRECTORY}/.versionbot/artefacts
+
+mv $(pwd)/artefacts/* $(pwd)/${ARGV_DIRECTORY}/.versionbot/artefacts || true
+
+pushd ${ARGV_DIRECTORY}
+
+baseOwner=$(jq -r '.base_org' .git/.version)
+baseRepo=$(jq -r '.base_repo' .git/.version)
+
+headBranch=$(jq -r '.head_branch' .git/.version)
+headOwner=$(jq -r '.head_org' .git/.version)
+headRepo=$(jq -r '.head_repo' .git/.version)
```

Comment:
I've seen this pattern in several scripts (`jq -r '.foo' .git/.version`). Maybe is time for us to refactor it to a lib. This is a reminder as a `to be discussed`, not something I would like to have fixed :)